doc/man3/X509_STORE_add_cert.pod

```diff
@@ -72,6 +78,11 @@ for the corresponding values used in certificate chain validation.  Their
 behavior is documented in the corresponding B<X509_VERIFY_PARAM> manual
 pages, e.g., L<X509_VERIFY_PARAM_set_depth(3)>.
 
+X509_STORE_add_lookup() finds or creates a L<X509_LOOKUP(3)> with the
+L<X509_LOOKUP_METHOD(3)> I<m> and adds it to the B<X509_STORE> I<v>.
```

Comment:
m and v are bad names; at least change m to method.

Reply:
This file is full of horrible names.  _ctx_ for a `X509_STORE`?  But that's what they're called in code...

But yeah, _m_ and _v_ are especially horrible.  Fixed